cms/utils/check.py

```diff
@@ -343,6 +343,35 @@ def get_class(method_name, model):
                                    'https://django-cms.readthedocs.io/en/latest/extending_cms/extending_page_title.html#handling-relations.')  # noqa
 
 
+@define_check
+def check_placeholder_mixin(output):
+    """
+    ModelAdmin instances that are using PlaceholderField fields
+    should be also a subclass of PlaceholderAdminMixin
+    """
+    from django.contrib.admin import site
+    from cms.models.fields import PlaceholderField
+    from cms.admin.placeholderadmin import PlaceholderAdminMixin
+
+    with output.section("PlaceHolderAdminMixin") as section:
+        for model, model_admin in site._registry.items():
+            ph_fields = [field for field in model._meta.get_fields() if isinstance(field, PlaceholderField)]
+            if len(ph_fields) == 0:
+                continue
+
+            if isinstance(model_admin, PlaceholderAdminMixin):
```

Comment:
I think is ok to remove this from output and instead only log if there's error.
so would result in:

```python
if not isinstance(model_admin, PlaceholderAdminMixin):
    ...
```